test_address_book.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import address_book
+from pathlib import Path
+
+
+class TestAddressBook(unittest.TestCase):
+    def setUp(self):
+        self.john = address_book.Contact("John", "email.address", "123")
+        self.ab = address_book.AddressBook('contact.json')
+        self.ab.add_contact(self.john)
+
+    def test_add_contact(self):
+        jack = address_book.Contact('Jack', 'jack@email.com', 234)
+        self.ab.add_contact(jack)
+        self.assertIn('John', self.ab._contacts)
```

Comment:
No need to check that John is in there here as you are testing the add_contact function and just added Jack